src/bp-forums/users/template.php

```diff
@@ -1410,23 +1410,16 @@ function bbp_get_user_topics_created_url( $user_id = 0 ) {
 		return false;
 	}
 
-	// Allow early overriding of the profile URL to cut down on processing
-	$early_url = apply_filters( 'bbp_pre_get_user_topics_created_url', (int) $user_id );
-	if ( is_string( $early_url ) ) {
-		return $early_url;
+	// Bail if intercepted
+	$intercept = bbp_maybe_intercept( 'bbp_pre_get_user_topics_created_url', func_get_args() );
+	if ( bbp_is_intercepted( $intercept ) ) {
+		return $intercept;
```

Comment:
@janakkaneriya007 you can revert this as we don't need interception code in the forums. it is used in bbpress's new version to handle filters.

Reply:
Fixed!!